### PR TITLE
React/dp 19539 footer slim multiple logos

### PR DESCRIPTION
**Pull request by clairesunstudio**
This PR:
- enables Footer Slim to render more than one logo with `stackedLogo` option
- removed site logo sizing constraint

Screen Shot 2020-08-14 at 2 31 35 PM
Screen Shot 2020-08-14 at 2 47 22 PM
 - removed 'moment' dependency from FooterSlim (see issue https://github.com/massgov/mayflower/issues/1140). 
> Note: This is not a full removal of `momentJS` in mayflower-react. InputDate is also dependent on the date functionalities, and we have plans to assess and switch to [date-fns](https://date-fns.org/) if neccessary

See PFML PR
https://github.com/EOLWD/pfml/pull/888

### CODE REVIEW

**Comment by clairesunstudio** [comment]
We need to figure out a way to link to the local mayflower-assets in circleCI tests, without having to publish the package and installing it to see run the tests

**Comment by clairesunstudio** [comment]
Not sure why backstop is throwing this error 
Screen Shot 2020-08-14 at 3 26 47 PM

File: react/src/components/organisms/FooterSlim/index.js [hunk 1]
```diff
@@ -16,60 +16,77 @@ import IconPhone from 'MayflowerReactBase/Icon/IconPhone';
...
-          <p>{props.description }</p>
-          <p className="ma__footer-slim__copyright">
-            &copy;
-            {' '}
-            {moment().year()}
-            {' '}
-            Commonwealth of Massachusetts
-          </p>
+const FooterSlim = ({
+  title, description, siteLogo, stackedLogo, links, contact
```
Comment (smurrayatwork) [inline react/src/components/organisms/FooterSlim/index.js]: `stackedLogo` needs a default value.

Comment (smurrayatwork) [reply react/src/components/organisms/FooterSlim/index.js]: Also `description`. Maybe:
```suggestion
  title = '', description = '', siteLogo, stackedLogo = false, links, contact = null
```

Comment (clairesunstudio) [reply react/src/components/organisms/FooterSlim/index.js]: I will add a default value to stackedLogo. but `title`, `description` are required props, so they won't need default props?

Comment (smurrayatwork) [reply react/src/components/organisms/FooterSlim/index.js]: That makes sense. 👍

**Comment by clairesunstudio** [comment]
replaced by #1208 with merged conflicts

**Pull request closed without merging**